### DISCUSSION TITLE
Support for both video & photos in same tweet

### DISCUSSION THREAD

**Discussion by dnestico** [robertodoering/harpy#663] [Ideas]
Twitter has been recently working on a feature that allows you to put both a video and photo in the same tweet, not sure if it's released just yet so this will be for future reference but would be cool if that was also possible on harpy, if the API allows it now of course.

<img width="600" alt="Screen Shot 2022-05-04 at 8 55 29 AM" src="https://user-images.githubusercontent.com/3034555/166685871-647f5cb7-8ab2-4690-915e-94dfb21641dd.png">

**Comment by FerasK0** [comment]
I have seen this on mastodon but weirdly some clients support it some clients don't.

For now we won't see it in harpy in v0.9.0 at least, since even Twitter themselves didn't implement it yet. I doubt that Roberto won't implement it if it's possible tho.
